pandas-stubs/core/reshape/pivot.pyi

```diff
@@ -33,12 +40,12 @@ def pivot(
     values: IndexLabel = ...,
 ) -> DataFrame: ...
 def crosstab(
-    index: Sequence | Series,
-    columns: Sequence | Series,
-    values: Sequence | None = ...,
-    rownames: Sequence | None = ...,
-    colnames: Sequence | None = ...,
-    aggfunc: Callable | None = ...,
+    index: list | npt.NDArray | Series | list[Sequence | npt.NDArray | Series],
+    columns: list | npt.NDArray | Series | list[Sequence | npt.NDArray | Series],
+    values: list | npt.NDArray | Series | None = ...,
```

Comment:
I think all 3 of these should be `AnyArrayLike | list[AnyArrayLike]`

Both `Index` and `Categorical` work for `index` and `columns`.  Didn't try `values`.

Also, if `values` is not `None`, then `aggfunc` can't be `None`, and vice versa (either both `None` or both are not `None`)

Reply:
These are all fixed now to allow the full set of AnyArrayLike | ArrayLike.